### ISSUE TITLE
Make image description/alt text for static visualizations non-mandatory

### ISSUE THREAD

**Issue by JoeHasell** [owid/owid-grapher#3201]
## Context

It is currently mandatory to set image descriptions to static images stored in GDocs before they can be added and published in an article. This is done to ensure that all images have alt-text.

## Problem

Authors commonly iterate on static images, and re-adding the alt text each time can be quite painful for them, as reported by Max.

## Impact

We know that having the alt text mandatory adds friction to the authoring flow. We have mixed intuitions about the costs/benefits of the alt text, we would love to experiment with a screen reader to see how valuable the alt text is.

## Proposed solution

1. Image descriptions/alt text should not be mandatory – an article should be publishable without one. As a middle ground, a warning should be raised in the settings draw when no text is provided.

We expect that it will then become more common for authors to make use of the override within the archie block itself. i.e. 

```
{.image}
filename: default-featured-image.png
alt: HERE
{}
```

2. In particular, get rid of this requirement for featured images. (It's hard to see the value of alt text for featured images – they aren't really doing anything than advertising the article, often in quite an abstract way.)

## Context

The problem was raised by Max, and the solution was proposed within a discussed between Lars, Max, Daniel and Joe.

## Technical notes

- Featured images are presentational, they shouldn't need alt text
  - We can use a special keyword in the alt text to mark them as skipped
- Static visualisations should have alt text or a warning

**Comment by toni-sharpe** [comment]
Are you aware/able to use `role='presentational`? 

I know I'm an outsider here, but I actually think the author should be doing the ALT as part of the work, just _only_ where they need to

What about - which would allow the user to skip that annoying bit, but still force them to think before doing that human of human things and doing the easy thing (like `any` in TS lol).

```
filename: default-featured-image.png
alt: PRES
```

Guessing a bit technically, but also trying to think of the problem as a whole

=====

**Some rationale**

I can see it would be annoying adding an alt for figure with a caption for example, or some design styling, but some things need `alt`. There's a position that the "significant pain" vasued to partially- and full un-sighted users would be greater than that of the author.

Also, given the idea that the knowledge should be as broadly avaialble as possible, bear in mind that in place with poorer connections ALT is often used whilke the image loads (which could take some minutes). AI might eliminate this problem soon.

A graph should never be presented visually, even in a non-ideallistic world I would argue this is proibably still still true, so one would look for more solid fallbacks, such as basic HTML tables.
